scylla/src/transport/topology.rs

```diff
@@ -873,6 +873,8 @@ async fn query_user_defined_types(
         .try_collect()
         .await?;
 
+    topo_sort_udts(&mut udt_rows)?;
```

Comment:
Small request: please measure the time it takes to toposort the UDT definitions and print it with `debug` level.